### PR TITLE
CSS Margin added

### PR DESCRIPTION
**Pull request by darshan4903**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #issue_number_here 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Web_Development/FrontEnd/CSS Margin/CSS Margin.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+There are properties for setting the margin for each side of an element (top, right, bottom, and left).
+## When to use this?
+When we need to shift any element from one position to another or when we need to create some space outside an element.
+## Margin property for Individual sides
+Margin property for each side of element:
+* For Top Margin:- `margin-top`
+* For Left Margin:- `margin-left`
+* For Bottom Margin:- `margin-bottom`
+* For Right Margin:- `margin-right`
+### 👉🏽Syntax
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/CSS Margin/CSS Margin.md:12]: Remove 👉🏽

File: Web_Development/FrontEnd/CSS Margin/CSS Margin.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  background-color: black;
+  margin: inherit;
+}
+body{
+  background-color: aqua;
+}
+```
+### References
+* https://www.w3schools.com/css/css_margin.asp
+* https://developer.mozilla.org/en-US/docs/Web/CSS/margin
```
Comment (kashishahuja2002) [inline Web_Development/FrontEnd/CSS Margin/CSS Margin.md:85]: Embed the link in the name of website.

**Review by kashishahuja2002 — CHANGES_REQUESTED** [review]

**Comment by darshan4903** [comment]
Updated @kashishahuja2002

**Review by kashishahuja2002 — APPROVED** [review]
LGTM.
Tag the supervisor of your batch to review and merge the PR.

**Comment by darshan4903** [comment]
@2024-SANDHYA please merge this PR

File: Web_Development/FrontEnd/CSS Margin/CSS Margin.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+# CSS Margin
+It can be used for creating space outside the element by this we can adjust the position of element like where we have to shift it.
+There are properties for setting the margin for each side of an element (top, right, bottom, and left).
```
Comment (2024-SANDHYA) [inline Web_Development/FrontEnd/CSS Margin/CSS Margin.md:3]: The sentence is not clear & understandable. Rephrase it. Also, don't use words like "you" "we".

**Review by 2024-SANDHYA — CHANGES_REQUESTED** [review]
Try to write clearly what you want to convey. Rest all is good !!

**Comment by darshan4903** [comment]
Updated @2024-SANDHYA

**Review by 2024-SANDHYA — APPROVED** [review]

**Pull request merged**